client/src/sections/quests/Level4.tsx

```diff
@@ -20,6 +20,11 @@ const Level4: React.FC = () => {
       minLevel={4}
       hide={step === Step.FINISHED}
     >
+      {step >= 0 && !have($item`enchanted bean`) && (
```

Comment:
Is it worth checking to see if you've already used the bean?

Reply:
yes it probably is, good call